src/app/component/followers/followers.component.html

```diff
@@ -0,0 +1,50 @@
+<div>
+  <h3 align="center">
+    🕸 My Followers
+  </h3>
+  <p align="center">You can see all the followers here👇</p>
+</div>
+
+<div class="clr-row clr-justify-content-center">
+<div *ngIf="(loadingFollowers)" class="clr-row clr-justify-content-center">
+  <div class="alert-items">
+      <div class="alert-item static clr-justify-content-center">
+          <span class="alert-text">Loading followers, please wait...</span>
```

Comment:
Follower generally means ones who follow the user.

If we say 'I am a follower of Mr.X', it means that i follow him, not that he follows me.

Here, we want to show people who we follow.